From c479a9412a11baad6f3b6075fa00b8d536ec3a4a Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Mon, 28 Mar 2016 11:19:44 -0500
Subject: [PATCH 1659/1691] MLK-12602: Documentation: gpmi-nand: legacy bch
 geometry option for NAND

commit c479a9412a11baad6f3b6075fa00b8d536ec3a4a from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

document the new option for legacy bch geometry support.

Conflicts:
	Documentation/devicetree/bindings/mtd/gpmi-nand.txt

Signed-off-by: Han Xu <han.xu@nxp.com>
(cherry picked from commit c1c24ecd24cb808e825eb13a3e3d016c283322cc)
---
 Documentation/devicetree/bindings/mtd/gpmi-nand.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
index d02acaf..c09f848 100644
--- a/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
+++ b/Documentation/devicetree/bindings/mtd/gpmi-nand.txt
@@ -35,6 +35,10 @@ Optional properties:
                        partitions written from Linux with this feature
                        turned on may not be accessible by the BootROM
                        code.
+ - fsl,legacy-bch-geometry: Use legacy bch geometry(ECC scheme) that
+                       compatible with 3.10 kernel. Without the property,
+                       software may use ECC strength according to NAND chip
+                       spec, e.g. ONFI standard.
 
 The device tree may optionally contain sub-nodes describing partitions of the
 address space. See partition.txt for more detail.
-- 
1.9.1

